server/src/main/java/org/apache/uniffle/server/storage/LocalStorageManager.java

```diff
@@ -61,6 +62,8 @@ public class LocalStorageManager extends SingleStorageManager {
   private final LocalStorageChecker checker;
   private List<LocalStorage> unCorruptedStorages = Lists.newArrayList();
   private final Set<String> corruptedStorages = Sets.newConcurrentHashSet();
+  private final ExecutorService executorService = Executors.newCachedThreadPool(
```

Comment:
We only use this thread pool to delete the files once when the server start. I think it's no need.

Reply:
Updated.